codalab/apps/web/templates/web/competitions/_results_page.html

```diff
@@ -88,6 +89,8 @@
                                     {{scoredata.username}}
                                 {% endif %}
                             </td>
+                            <td> {{scoredata.count}} </td>
```

Comment:
annoying nitpick incoming: will you pull the spaces around this and put spaces inside the braces so it looks like... `<td>{{ scoredata.count }}</td>`